x/group/types.go

```diff
@@ -152,6 +152,7 @@ func (g GroupMember) PrimaryKey() []byte {
 }
 
 func (g GroupAccountInfo) PrimaryKey() []byte {
+	// TODO: the following line fails, because sdk.AccAddressFromBech32 requires 20-byte addresses .
 	addr, err := sdk.AccAddressFromBech32(g.Address)
```

Comment:
here we have a bigger problem. 
`g.Address` is 32 bytes address. `AccAddressFromBech32` doesn't` accept that addresses and return errors.
Proposed solution:
* update `sdk.AccAddressFromBech32`

Reply:
solved this by refactoring the `rootModuleKey`